test/Faker/Container/ContainerTest.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Faker\Container;
+
+use Faker\English\FileExtension;
+use Faker\Test\TestCase;
+use Psr\Container\ContainerExceptionInterface;
+use Psr\Container\NotFoundExceptionInterface;
+
+final class ContainerTest extends TestCase
+{
+    public function testGetNonExisting()
+    {
+        $container = new Container(['file' => FileExtension::class]);
+        self::assertFalse($container->has('foo'));
+        $this->expectException(NotFoundExceptionInterface::class);
+        $container->get('foo');
+    }
+
+    public function testGetFromString()
```

Comment:
How do you feel about adjusting the names of test methods to reveal what they are testing?

I think it's always a good idea to keep in mind the potential output from running tests with the [Testdox printer](https://phpunit.readthedocs.io/en/9.5/textui.html?highlight=testdox#testdox).

Reply:
I thought I did... This test if I can get a service from a string definition.